stream-chat-android-state/src/main/java/io/getstream/chat/android/state/event/handler/internal/EventHandlerSequential.kt

```diff
@@ -230,6 +232,25 @@ internal class EventHandlerSequential(
         scope.coroutineContext.job.cancelChildren()
     }
 
+    private fun handleUserMessagesDeletedEvent(event: UserMessagesDeletedEvent) {
```

Comment:
Given the recent comments and the coverage of this PR, maybe you need to create some unit tests here to increase it and be at least closer to 70%.
WDYT?